.github/workflows/triage.yml

```diff
@@ -18,25 +29,26 @@ jobs:
             // - core: A reference to the @actions/core package
             // - io: A reference to the @actions/io package
 
-            // Check if issue has a JIT label.
-            const kJitLabel = "oncall: jit";
+            const kName = process.env.NAME;
+            // Check if issue has the relevant label.
+            const kLabel = `oncall: ${kName.toLowerCase()}`;
 
             issue = await github.issues.get({
               owner: context.issue.owner,
               repo: context.issue.repo,
               issue_number: context.issue.number,
             })
 
-            const hasJitLabel = issue.data.labels.filter(label => label.name == kJitLabel).length > 0;
+            const hasLabel = issue.data.labels.filter(label => label.name == kLabel).length > 0;
 
-            if (!hasJitLabel) {
-              core.debug("Issue " + issue.data.title + " does not have JIT label");
+            if (!hasLabel) {
+              core.debug(`Issue ${issue.data.title} does not have ${kName} label`);
               return;
             }
 
             // Get project column ID.
-            const kProjectName = "JIT Triage";
-            const kColumnName = "Need triage";
+            const kProjectName = `${kName} Triage`;
```

Comment:
Just curious, why is the convention to have k's starting every variable name?

Reply:
@janeyx99 Some coding styles prefix `const`ant names with `k`. Before this PR, all these variables actually are literal constant strings, but I guess after this PR, their values are determined by the matrix so they're not really constants; perhaps I should have removed the `k` prefix.